shared-components/tl-tournament-countdown.js

```diff
@@ -0,0 +1,111 @@
+import { LitElement, html, css } from "lit";
+
+export class TournamentCountdown extends LitElement {
+    static properties = {
+        dateStart: {
+            type: "number",
+            attribute: "date-start",
+        },
+        dateEnd: {
+            type: "number",
+            attribute: "date-end",
+        },
+        _status: { state: true },
+        _total: { state: true },
+        _days: { state: true },
+        _hours: { state: true },
+        _minutes: { state: true },
+        _seconds: { state: true },
+    };
+
+    static get styles() {
+        return [
+            css`
+                .countdown {
+                    position: absolute;
```

Comment:
This is an issue to me: wherever we put the component it is going to be move to the top of it's parent!!! And there's no possibility to fix it from outside...

You should rely on the developer inserting your component in the DOM to decide for its position...

As described in another comment, I would have placed the countdown component before the image, and have the image with a negative `margin-top` so it would have gone below the countdown component...